source/index-management.rst

```diff
@@ -185,6 +204,44 @@ Standard API
     listIndexes(): Cursor;
   }
 
+  interface CreateIndexOptions {
+    /**
+     * Specifies how many data-bearing members of a replica set, including the primary, must
+     * complete the index builds successfully before the primary marks the indexes as ready.
+     *
+     * This option accepts the same values for the "w" field in a write concern plus "all",
+     * which indicates all voting data-bearing nodes.
+     *
+     * @note This option is only available in MongoDB 4.4+.
```

Comment:
nit: tags like this are usually put at the bottom of the docblock

Reply:
also I think a better tag to use would be [@since](https://jsdoc.app/tags-since.html)

Reply:
done and done